recipes/diligent-fx/all/patches/0001-use-conan-dependencies.diff

```diff
@@ -0,0 +1,18 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index 7d56e79..46b1113 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -18,11 +18,8 @@ endif()
+ 
+ target_link_libraries(DiligentFX 
+ PRIVATE
+-    Diligent-BuildSettings
+-PUBLIC
+-    Diligent-GraphicsEngine
+-    Diligent-GraphicsTools
+-    Diligent-AssetLoader
++    CONAN_PKG::diligent-core
++    CONAN_PKG::diligent-tools
```

Comment:
Better to use actual targets if they are available (maybe need to run `find_package()` before) because these targets won't be available in the future. Anyway, so far, so good, when we change the generator to build this library we can change here as well.

Reply:
already answered here.

Sure, I will use CONAN_PKG::